File: less/mario.c
#include <stdio.h>
#include<cs50.h>

int main(){
	int n;
	do
	{
	    n = get_int("Size :");
	}while(n<=0 || n>8);

	for (int i = 1; i<=n; i++){
		for (int j = 1; j<=n-i; j++){
			printf(" ");
		}
		for (int j = 1; j<=i; j++){
			printf("#");
		}
		printf("\n");
	}

	return 0;
}

File: sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

def main():

    while True:
        n = get_int("Height : ")
        if n >= 1 and n <=8:
            break

    for i in range(n):
        print(" " *(n-1-i),end="")
        print("#" *(i+1),end="")
        print(" " *2,end="")
        print("#" *(i+1),end="")
        print(end="\n")

if __name__ == "__main__":
    main()